index.html

```diff
@@ -3,48 +3,55 @@
+            <button class="favoriteStyled clothes-Options tuxedo margin-left-class-btn" type="button">Tuxedo</button>
+        </div>
         </article>
-        <article class="accessories-selection">
-          <h3 class= clothes-title>Accessories</h3>
-          <button class="accessories-Options neckalace" type="button">Neckalace</button>
-          <button class="accessories-Options bowtie" type="button">Bowtie</button>
-          <button class="accessories-Options watch" type="button">Watch</button>
-          <button class="accessories-Options monocle" type="button">Monocle</button>
-          <button class="accessories-Options earrings" type="button">Earrings</button>
+        <article class="accessories-selection" id="accessories-selection">
+          <div class="accessories-header-btns">
+            <h3 clothes-title><span class="arrows">➫</span>Accessories</h3>
+            <button class="favoriteStyled accessories-Options neckalace" type="button">Necklace</button>
+            <button class="favoriteStyled accessories-Options bowtie margin-left-class-btn" type="button">Bowtie</button>
+            <button class="favoriteStyled accessories-Options watch margin-left-class-btn" type="button">Watch</button>
+            <button class="favoriteStyled accessories-Options monocle" type="button">Monocle</button>
+            <button class="favoriteStyled accessories-Options earrings margin-left-class-btn" type="button">Earrings</button>
+        </div>
         </article>
-        <article class="background-selection">
-          <h3 class= clothes-title>Backgrounds</h3>
-          <button class="background-Options blue" type="button">Blue</button>
-          <button class="background-Options park" type="button">Park</button>
-          <button class="background-Options beach" type="button">Beach</button>
-          <button class="background-Options space" type="button">Space</button>
-          <button class="background-Options yellow" type="button">Yellow</button>
-          <button class="background-Options hearts" type="button">Hearts</button>
+        <article class="background-selection" id="background-selection">
+          <div class="background-header-btns">
+            <h3 class= clothes-title><span class="arrows">➪</span>Backgrounds</h3>
+            <button class="favoriteStyled background-Options blue" type="button">Blue</button>
+            <button class="favoriteStyled background-Options park margin-left-class-btn" type="button">Park</button>
+            <button class="favoriteStyled background-Options beach margin-left-class-btn" type="button">Beach</button>
+            <button class="favoriteStyled background-Options space margin-left-class-btn" type="button">Space</button>
+            <button class="favoriteStyled background-Options yellow" type="button">Yellow</button>
+            <button class="favoriteStyled background-Options hearts margin-left-class-btn" type="button">Hearts</button>
```

Comment:
Section that have multiple classes name like this section will need to be worked on where we can condense the length by using one or two class so that we don't use any unnecessary class name that increase the overall length of code.